kratos/geometries/geometry.h

```diff
@@ -3189,20 +3192,53 @@ class Geometry
 
 
 private:
-    ///@name Static Member Variables
-    ///@{
-
-    // static const GeometryData msEmptyGeometryData;
-
-    ///@}
     ///@name Member Variables
     ///@{
 
+    IndexType mId;
```

Comment:
The type should be `uint64_t` to be sure that it works also in 32bits machines.

Reply:
I have some questions to this:
- do we need to change the indexed_object accordingly?
- does that mean that we need to update the IndexType accordingly, or start using a IdType?